src/dmd/e2ir.d

```diff
@@ -4809,12 +4806,12 @@ elem *toElem(Expression e, IRState *irs)
                     //  lwr <= upr
 
                     elem *c1 = null;
-                    elem *elen;
```

Comment:
dmd/e2ir.d(4854): Error: undefined identifier `elen`, did you mean struct `elem`?

Reply:
fixed